execution_engine_testing/tests/Cargo.toml

```diff
@@ -39,18 +39,3 @@ wat = "1.0.47"
 [features]
 use-as-wasm = ["casper-engine-test-support/use-as-wasm"]
 fixture-generators = []
-
-[lib]
-bench = false
-
-[[bench]]
-name = "transfer_bench"
-harness = false
-
-[[bench]]
-name = "auction_bench"
```

Comment:
Is there a plan to revive these benches later?

Reply:
Not that I know of.